.github/workflows/aws_tfhe_gpu_tests.yml

```diff
@@ -97,6 +97,15 @@ jobs:
         run: |
           make test_gpu
 
+      - name: Run internal CUDA tests
+        run: |
+          cd backends/tfhe-cuda-backend/cuda/
+          mkdir build
+          cd build
+          cmake -DTFHE_CUDA_BACKEND_BUILD_TESTS=ON ../
+          make -j8
+          tests_and_benchmarks/tests/test_tfhe_cuda_backend
```

Comment:
There is a command in the Makefile to do this now, and when doing `make test_gpu` it launches custom cuda tests as well. So this block is not necessary.